rotkehlchen/tests/unit/test_globaldb.py

```diff
@@ -988,3 +989,31 @@ def test_general_cache(globaldb):
             key_parts=[GeneralCacheType.CURVE_POOL_TOKENS, '123'],
         )
         assert values_8 == values_1
+
+
+def test_packaged_db_check_for_constant_assets(globaldb):
+    """
+    Check if UnknownAsset is raised for an asset in CONSTANT_ASSETS, the packaged global db
```

Comment:
in any tests you write, you have to also check for `WrongAssetType` error and you also have to make sure that later it has also been written in the actual global DB, what I asked you in the previous file